source/dub/commandline.d

```diff
@@ -1815,7 +1820,19 @@ class UpgradeCommand : Command {
 			"",
 			"This will update the versions stored in the selections file ("~SelectedVersions.defaultFile~") accordingly.",
 			"",
-			"If one or more package names are specified, only those dependencies will be upgraded. Otherwise all direct and indirect dependencies of the root package will get upgraded."
+			"If one or more package names are specified, only those dependencies will be upgraded. Otherwise all direct and indirect dependencies of the root package will get upgraded.",
+			"",
+			"List output format: (not script-safe)",
```

Comment:
I wouldn't write "(not script-safe)". Except for `dub describe`, nothing is script safe.